locales/en/settings.ftl

```diff
@@ -84,6 +84,6 @@ settings-unsubscribe-dialog-title = We’re sorry to see you go. <br /> Will you
 settings-unsubscribe-dialog-info = Your experience is important to us. We read every response and take it into consideration.
 settings-unsubscribe-dialog-message-placeholder = What could have gone better?
 # $faq_href is the URL for the faq page. HTML tags should not be translated, e.g. `<a>`
-settings-unsubscribe-dialog-confirmation = Please note, all of your { -brand-monitor-premium } services will be <a { $faq_href }>permanently deleted</a> after your current billing cycle ends.
+settings-unsubscribe-dialog-confirmation = Please note, all of your { -brand-monitor-plus } services will be <a { $faq_href }>permanently deleted</a> after your current billing cycle ends.
```

Comment:
You need a new string ID to make this type of change.
https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html